parlai/nn/lr_scheduler.py

```diff
@@ -87,12 +98,17 @@ def load_state(self, states):
         """
         Load state of scheduler from states.
         """
-        if self.scheduler and 'lr_scheduler' in states:
-            self.scheduler.load_state_dict(states['lr_scheduler'])
         if states.get('warmup_scheduler') and getattr(self, 'warmup_scheduler', None):
             self.warmup_scheduler.load_state_dict(states['warmup_scheduler'])
+        if self.scheduler and 'lr_scheduler' in states:
```

Comment:
is there any reason for moving this?

Reply:
Ya idk something crashed for me but I don't remember.